apps/system/init/init.c

```diff
@@ -67,7 +67,7 @@ static void tash_register_cmds(void)
 	kernel_register_utilcmds();
 #endif
 
-#ifdef CONFIG_FS_CMDS
+#if !defined(CONFIG_BUILD_PROTECTED) && defined(CONFIG_FS_CMDS)
```

Comment:
@KimDongEon Could you share the schedule of fixing fs cmds?

Reply:
I will merge it into this week.